core/ui/src/components/settings/CloudLogManagerConfigureModal.vue

```diff
@@ -17,6 +17,12 @@
     }}</template>
     <template slot="content">
       <cv-form v-if="subscription">
+        <NsInlineNotification
+          v-if="this.cluster_id != ''"
```

Comment:
- Let's use `v-if="clusterId"`. The condition `this.cluster_id != ''` should be fine but might be too specific: what if `cluster_id == undefined` instead of empty string?
- `this` is not needed inside templates